waspc/src/Wasp/Generator/SdkGenerator/AuthG.hs

```diff
@@ -115,10 +115,7 @@ genUtils auth = return $ C.mkTmplFdWithData relUtilsFilePath tmplData
 
 genIndexTs :: AS.Auth.Auth -> Generator [FileDraft]
 genIndexTs auth =
-  return $
-    if isEmailAuthEnabled || isLocalAuthEnabled
```

Comment:
`wasp/auth` was needed in the case of Github and Google auth as well, fixes the logic